docker/Dockerfile.test

```diff
@@ -0,0 +1,28 @@
+FROM ghcr.io/woodard/libabigail-fedora-base
+
+# docker build -f docker/Dockerfile.test -t test .
+
+RUN dnf install -y \
+    file \
+    diffutils \
+    elfutils-libs \
```

Comment:
I think that these shouldn't be necessary here because we have the devel packages in the build environment. I may have lost the control logic though. If they are needed here then there are a few more runtime libs needed.

Reply:
We would have only elfutils-libs in the fedora base container, but not the other two (since we are removing them). I'm adding them all here because we haven't built -> deployed the base container with these changes yet.